store/rootmulti/store.go

```diff
@@ -148,9 +148,9 @@ func (rs *Store) GetCommitKVStore(key types.StoreKey) types.CommitKVStore {
 	return rs.stores[key]
 }
 
-// GetStores returns mounted stores
-func (rs *Store) GetStores() map[types.StoreKey]types.CommitKVStore {
-	return rs.stores
+// GetStoreNames returns the storeNames and StoreKeys
+func (rs *Store) GetStoreNames() map[string]types.StoreKey {
```

Comment:
Why renaming? If we want to rename then let's be more precise. I suggest: `StoreKeysByName` (no need for _Get_ prefix)